tools/walletextension/wallet_extension.go

```diff
@@ -54,20 +46,11 @@ const (
 	corsAllowHeaders = "Access-Control-Allow-Headers"
 	corsHeaders      = "Accept, Content-Type, Content-Length, Accept-Encoding, X-CSRF-Token, Authorization"
 
-	Localhost         = "127.0.0.1"
-	websocketProtocol = "ws://"
+	Localhost = "127.0.0.1"
```

Comment:
@BedrockSquirrel This can move over to the `main` package, since that's it's only usage.